schema/me/lot_standing.js

```diff
@@ -54,14 +64,10 @@ export default {
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  resolve: (root, { sale_id, artwork_id }, request, { rootValue: { accessToken } }) =>
-    Promise.all([
-      gravity.with(accessToken)("me/lot_standings", {
-        sale_id,
-        artwork_id,
-      }),
-    ]).then(([lotStanding]) => {
-      if (lotStanding.length === 0) return null
-      return lotStanding[0]
-    }),
+  resolve: (root, { sale_id, artwork_id }, request, { rootValue: { lotStandingLoader } }) => {
+    if (!lotStandingLoader) return null
+    return lotStandingLoader({ sale_id, artwork_id }).then(([lotStanding]) => {
```

Comment:
Nice, at first I was confused about `lotStanding` vs `lotStanding[0]`, but then I realized the old code was using `Promise.all` (for some reason, even though there was just one promise)?

Reply:
Yeah not totally sure why that was there but this seems to work just as well 🤷‍♀️